docsite/docs/Docs/fal-cli/pure-python-models.md

```diff
@@ -0,0 +1,129 @@
+---
+sidebar_position: 8
+---
+
+# Making pure Python dbt models
+
+In this guide we will build a dbt model with a Python script using fal.
+
+This guide is inspired by Andy Reagan's post where fal is mentioned, [Non-SQL in your DBT pipeline](https://andyreagan.medium.com/non-sql-in-your-dbt-pipeline-c7cef2091619). Which in turn is inspired by Claire Carroll's post on the dbt Discourse, [Representing non-SQL models in a dbt DAG](https://discourse.getdbt.com/t/representing-non-sql-models-in-a-dbt-dag/2083).
+
+<!-- TODO: Why would we do this? -->
+We love dbt, but it only runs SQL, some stuff can only (or much more easily) be achieved in other programming languages. In this case, we are going to use Python.
+
+## Context
+
+We are going to expand the jaffle shop classic example of dbt. We will add a life time value prediction to the shop costumers.
+
+## Make dbt _know_ about the model
+
+The first thing we want to do is to actually create a dbt model. 
+
+```sql
+{{ config(materialized='ephemeral') }}
+/*
+This table is generated by fal.
+
+Please run this project with `fal flow` CLI tool instead of `dbt` to
+get the correct results.
+
+Specify script dependencies here for dbt and fal to know dependencies:
+
+{{ ref('customers') }}
+{{ ref('orders') }}
+*/
+
+-- Check Python file for real behaviour
+SELECT * FROM {{ target.schema }}.{{ model.name }}
+```
+> File: `models/customers_predicted_ltvs.sql`
+
+Things to note:
+
+- Including the `ref`s in a comment means that the edges in your DAG will correctly drawn between other models and this non-SQL model
+- Declaring this as an `ephemeral` model means dbt will not build this model
+- Now this can be documented with `descriptions` in the `schema.yml` file
+
+And let's add it to the schema.
```

Comment:
I don't think we need to show `schema.yml` twice. It's better to show it with the script attached, in it's final state.